src/server/editorServices.ts

```diff
@@ -447,6 +459,26 @@ namespace ts.server {
             });
         }
 
+        private loadTypesMap() {
+            if (!this.host.fileExists(this.typesMapLocation)) {
+                this.logger.info(`Provided types map file "${this.typesMapLocation}" doesn't exist`);
+                return;
+            }
+            try {
+                const raw: TypesMapFile = JSON.parse(this.host.readFile(this.typesMapLocation));

```

Comment:
Ah, nevermind. I just saw that you want to port this to 2.3, which doesn't have these two methods.